std/haxe/io/Bytes.hx

```diff
@@ -539,6 +539,22 @@ class Bytes {
 		return new Bytes(b.length,b);
 		#end
 	}
+	
+	public static function ofHex( s : String ) : Bytes {
+		var len:Int = s.length;
+		if ( (len & 1) != 0 ) throw "Hex string have a odd number of digits";
```

Comment:
Why do some targets throw string while others throw `Error.OutOfBounds`?
Let's be consistent and throw "Hex string has an odd number of digits" on all targets.

Reply:
Nitpick: If it has an odd number of digits, it's by definition not a hex string.

Reply:
It's still better to print the real reason of an exception :)

Reply:
Yes but then we should go for something like "Not a hex string (odd number of digits)".

Reply:
I agree.

Reply:
> If it has an odd number of digits, it's by definition not a hex string.

FYI, Javascript does not agree. On encoding, it is happy to create odd-length string. On decoding, it is happy to assume a leading `0`:
![image](https://user-images.githubusercontent.com/2192439/45170300-cd3b8c80-b1bc-11e8-8d0e-b4a34cfe2bae.png)

Reply:
Meh, I find that weird because it's not clear to me if that's ""04d2" or 4d20".